pkg/config/config.go

```diff
@@ -15,10 +15,14 @@
 package config
```

Comment:
do we need a new pkg for this?
Maybe we could move template related info to entities.Set, and registry related const to pkg/registry.

Reply:
Make sense. Thanks!